reflex-dom-core/src/Reflex/Dom/Builder/Class/Events.hs

```diff
@@ -159,7 +159,17 @@ type family EventResultType (en :: EventTag) :: * where
   EventResultType 'TouchmoveTag = TouchEventResult
   EventResultType 'TouchendTag = TouchEventResult
   EventResultType 'TouchcancelTag = TouchEventResult
-  EventResultType 'WheelTag = ()
+  EventResultType 'WheelTag = WheelEventResult
+
+data DeltaMode = DeltaPixel | DeltaLine | DeltaPage
```

Comment:
`Bounded, Enum` too?

Reply:
`Generic` while we're at it?

Reply:
@alexfmpe Everything is missing `Generic` and `Typeable`, so I suppose we should get those later?